### PR TITLE
Bump kubetest + kubekins-e2e

### PR DESCRIPTION
**Pull request by ixdy**
This pulls in #6846, #6861, #6864, #6870, and maybe other things.

cc @porridge @shyamjvs @dims @filbranden

/assign @BenTheElder

### CODE REVIEW

File: scenarios/kubernetes_e2e.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 ORIG_CWD = os.getcwd()  # Checkout changes cwd
 
 # Note: This variable is managed by experiment/bump_e2e_image.sh.
-DEFAULT_KUBEKINS_TAG = 'v20180212-83c830734-master'
+DEFAULT_KUBEKINS_TAG = 'v20180216-90c54ef53-master'
```
Comment (krzyzacy) [inline scenarios/kubernetes_e2e.py]: lol we won't need this anymore, I'll clean it up in a bit

**Comment by BenTheElder** [comment]
/lgtm

Probably will do some more bumping soon for experimental images, today is the day of image bumping :joy:

**Comment by BenTheElder** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-canary/11962
once this is green should be good to merge

**Comment by BenTheElder** [comment]
Green run above `I0216 21:54:32.555] Image: gcr.io/k8s-testimages/kubekins-e2e:v20180216-90c54ef53-master`

/hold cancel

**Pull request merged**